xla/service/buffer_assignment.cc

```diff
@@ -1952,8 +1965,8 @@ absl::Status BufferAssigner::AssignBuffersWithSequentialOrdering(
                                assignment->module(), schedule,
                                assignment->alias_analysis(), alias_info_,
                                assignment->buffer_size_, options));
-        AssignBuffersFromHeapSimulator(result, assignment, color,
-                                       isolation_options);
+        std::ignore = AssignBuffersFromHeapSimulator(result, assignment, color,
```

Comment:
Same here, we should preserve the previous behavior and CHECK fail if the returned status here is not ok.